azure_functions_worker/dispatcher.py

```diff
@@ -273,6 +276,7 @@ async def _handle__worker_init_request(self, request):
                     )
 
         worker_init_request = request.worker_init_request
+        directory = worker_init_request.function_app_directory
```

Comment:
nit: change to func_app_directory for more details?

Reply:
Really dont have a preference here. Named it directory to keep it consistent in all __handle methods since I think metadata requests has as directory